exercises/exercise3.js

```diff
@@ -1,12 +1,56 @@
 // Set this assignment to true if you do want to use it.
-module.exports.ACTIVATE_BOT = false;
+module.exports.ACTIVATE_BOT = true;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
+    label: 'Would you rather!',
+    prompt: 'Would you rather go shopping or out to eat?',
     handler: function () {
-      return 'Good Evening to You!';
+      return 'its 2020, solid food is for the weak! lets go shopping';
     },
+    isCaseSensitive: false,
+    isReply: true,
+    isLitening: true
   },
+
+  {
+    label: 'Would you rather the aliens that make first contact be organics or robots',
+     prompt: 'I would rather #{name} make first contact. ',
```

Comment:
...will want to take out that extra space after '...contact. '. Unfortunately it won't trigger the prompt with that in there.